grpc-server/src/main/resources/application.conf

```diff
@@ -1,4 +1,10 @@
-waves.grpc {
-  host = localhost
-  port = 6870
+waves {
+  grpc {
+    host = localhost
+    port = 6870
+  }
+
+  blockchain-updates {
```

Comment:
Shouldn't this setting be inside `grpc` because it's the same extension?
```
waves {
  grpc {
    host = localhost
    port = 6870
    blockchain-updates {
      port = 6881
...
```

Reply:
Actually they're not. Both are packaged in the same tarball/package, but are quite independent.